editor/editor_node.cpp

```diff
@@ -1291,7 +1291,7 @@ void EditorNode::_dialog_action(String p_file) {
 			uint32_t current = editor_history.get_current();
 			Object *current_obj = current > 0 ? ObjectDB::get_instance(current) : NULL;
 
-			ERR_FAIL_COND(!current_obj->cast_to<Resource>())
```

Comment:
This doesn't seem right, plus it would conflict with #10581.

Basically, `->cast_to` was assumed to run on null pointers as well, and it would return null if (1) `this == NULL`, or (2) `this` couldn't be casted to the specified type.